### PR TITLE
Remove the Basic JS example

### LINKED ISSUE #27: Remove the Basic JS example
The current choice of project types to create may be confusing for beginners:

* Basic JS App
* ES6 App (using Babel transpiler)
* TypeScript App

With iOS 8 and 9 being the last Tabris.js platforms that don't support ES6, and those are quickly losing market share, it seems to be a good time to move forward to ES6 and not advertise ES5 anymore.

I'd suggest to reduce the project types to:

* JavaScript App
* TypeScript App

The JavaScript App should be based on ES6. If we use `require` instead of `import`, we should not need a transpile step anymore.

### PR DESCRIPTION
**Pull request by camstuart**
Now that an ES6 syntax style has been found, it is no longer
necessary to have a 'Basic JS' app and and ES6 app

- Removed all traces of the ES6 App type option
- Updated the Basic JS app to use the new ES6 syntax

Resolves #27

### CODE REVIEW

File: app/templates/_package.json [hunk 1]
```diff
@@ -4,14 +4,10 @@
...
-    "build:ios": "BABEL_ENV=ios npm run build",
-    "build:android": "BABEL_ENV=android npm run build",
-    "build:win": "BABEL_ENV=win npm run build",
-    "build": "rm -rf dist/ && babel --compact false --out-dir dist/ src/",
-<% } else if (proj_type === 'ts') { %>
-    "build": "tsc -p .",
+<% if (proj_type === 'ts') { %>
+  "build": "tsc -p ."
+<% } else  { %>
+  "start": "tabris serve"
```
Comment (ralfstx) [inline app/templates/_package.json]: Shouldn't the start command should be available for both types?

Comment (camstuart) [inline app/templates/_package.json]: Hmm, yes I suppose it should. I'm not at all familiar with Type Script. 

So we would need build and start I imagine?

Comment (ralfstx) [inline app/templates/_package.json]: Yes, exactly. `build` only for ts and `start` for both.

Comment (camstuart) [inline app/templates/_package.json]: Done, amendment pushed

File: app/templates/_package.json [hunk 2]
```diff
@@ -4,14 +4,9 @@
...
   "scripts": {
-<% if (proj_type === 'es6') { %>
-    "build:ios": "BABEL_ENV=ios npm run build",
-    "build:android": "BABEL_ENV=android npm run build",
-    "build:win": "BABEL_ENV=win npm run build",
-    "build": "rm -rf dist/ && babel --compact false --out-dir dist/ src/",
-<% } else if (proj_type === 'ts') { %>
-    "build": "tsc -p .",
+<% if (proj_type === 'ts') { %>
+  "build": "tsc -p .",
```
Comment (cpetrov) [inline app/templates/_package.json]: Can you indent this line with four spaces?

File: app/templates/_package.json [hunk 3]
```diff
@@ -4,14 +4,9 @@
...
 <% } %>
-    "start": "tabris serve"
+  "start": "tabris serve"
```
Comment (cpetrov) [inline app/templates/_package.json]: Can you indent this line with four spaces?

Comment (camstuart) [inline app/templates/_package.json]: No worries, amended.

**Pull request merged**